packages/react/src/components/ActivityItem/__snapshots__/ActivityItem.test.tsx.snap

```diff
@@ -16,10 +16,19 @@ exports[`ActivityItem renders compact with a single persona correctly 1`] = `
         font-weight: 400;
         justify-content: flex-start;
       }
+      {
+        align-items: center;
```

Comment:
These changes are because previously the `mergeStyles` approach would allow users to pass in both `root` and `isCompactRoot`, then merge those styles. That's not possible with `styled()`, so the element in question receives both classNames/style blocks, though the end result is the same.